lib/src/ktfmt/java/com/diffplug/spotless/glue/ktfmt/KtfmtFormatterFunc.java

```diff
@@ -72,30 +67,17 @@ private FormattingOptions createFormattingOptions() throws Exception {
 			formattingOptions = Formatter.KOTLINLANG_FORMAT;
 			break;
 		default:
-			throw new IllegalStateException("Unknown formatting option");
+			throw new IllegalStateException("Unknown formatting option " + style);
 		}
 
 		if (ktfmtFormattingOptions != null) {
-			try {
-				formattingOptions = formattingOptions.copy(
-						formattingOptions.getStyle(),
-						ktfmtFormattingOptions.getMaxWidth().orElse(formattingOptions.getMaxWidth()),
-						ktfmtFormattingOptions.getBlockIndent().orElse(formattingOptions.getBlockIndent()),
-						ktfmtFormattingOptions.getContinuationIndent().orElse(formattingOptions.getContinuationIndent()),
-						ktfmtFormattingOptions.getRemoveUnusedImport().orElse(formattingOptions.getRemoveUnusedImports()),
-						formattingOptions.getDebuggingPrintOpsAfterFormatting(),
-						formattingOptions.getManageTrailingCommas());
-			} catch (NoSuchMethodError e) {
-				//noinspection JavaReflectionMemberAccess, ABI change from ktfmt 0.47
-				Method copyMethod = formattingOptions.getClass().getMethod("copy", FormattingOptions.Style.class, int.class, int.class, int.class, boolean.class, boolean.class);
-				formattingOptions = (FormattingOptions) copyMethod.invoke(formattingOptions,
-						formattingOptions.getStyle(),
-						ktfmtFormattingOptions.getMaxWidth().orElse(formattingOptions.getMaxWidth()),
-						ktfmtFormattingOptions.getBlockIndent().orElse(formattingOptions.getBlockIndent()),
-						ktfmtFormattingOptions.getContinuationIndent().orElse(formattingOptions.getContinuationIndent()),
-						ktfmtFormattingOptions.getRemoveUnusedImport().orElse(formattingOptions.getRemoveUnusedImports()),
-						formattingOptions.getDebuggingPrintOpsAfterFormatting());
-			}
+			formattingOptions = formattingOptions.copy(
+					ktfmtFormattingOptions.getMaxWidth().orElse(formattingOptions.getMaxWidth()),
+					ktfmtFormattingOptions.getBlockIndent().orElse(formattingOptions.getBlockIndent()),
+					ktfmtFormattingOptions.getContinuationIndent().orElse(formattingOptions.getContinuationIndent()),
+					formattingOptions.getManageTrailingCommas(),
+					ktfmtFormattingOptions.getRemoveUnusedImport().orElse(formattingOptions.getRemoveUnusedImports()),
+					formattingOptions.getDebuggingPrintOpsAfterFormatting());
```

Comment:
Same reason as the above.

Reply:
Should update all related changes.

Reply:
That's already handled. See `KtfmtStep.KtfmtFormatterFuncCompat`